README.rst

```diff
@@ -101,7 +79,7 @@ Check the `example`_ folder for some sample usage!
     environment variables.
 
     Variables are shallowly resolved once, then anything in
-    {`...`} is eval'd in Python.
+    {%...%} is eval'd in Python.
```

Comment:
This entire clause is going to be taken out from the `--help`, by the way, so it's fine to just delete it.